www/index.html

```diff
@@ -209,6 +209,17 @@ <h3 id="searchSyntax">Title search usage</h3>
                     </p>
                     <p style="text-align: right"><a href="#contents">↑ Back to Contents</a></p>
                     
+                    <h3 id="imageDownload">Image download</h3>
+                    <p>
+                        It's unfortunately not possible to download an image (from a page of a ZIM file) with the standard feature of browsers (usually right-click -> "Save
+                        image as..."). See <a href="https://github.com/kiwix/kiwix-js/issues/694">issue 694</a>.
+                    </p>
+                    <p>
+                        As a workaround, it's possible to choose "Copy Image" in the context menu (after the right-click), then paste it in any suitable software (Paint, Gimp
+                        etc, or even a Word processor), then save it.
```

Comment:
Slight revision to indicate that the workaround is for browser extensions (e.g. right-click doesn't work in UWP apps):

> One workaround (at least in browser extensions) is to choose "Copy Image" in the context menu (after right-click or long-press)...